### DISCUSSION TITLE
Rad Pro: Background compensation

### DISCUSSION THREAD

**Discussion by J0s3f** [Gissio/radpro#112] [Q&A] [answered]
I can see that the background compensation is configurable up to 0.015 µSv/h

But descriptions for the J321 tube (e.g. [here](https://a.aliexpress.com/_EyO7iuX)) show up to 25cpm background.
My understanding is, with a factor of 153.8cpm/µSv/h, this could be up to 25/153.8=0.163 µSv/h

Is something wrong in my thought process, or is there another good reason for compensation being limited to 0.015µSv/h?

**Comment by DG0MG** [comment, marked as answer]
The reason is the different meanings of the word "background". The technical data of the Chinese counter tubes mean that the specified pulse rate (e.g. 80 cpm) is to be expected with "normal" background radiation (i.e. somewhere around 0.05-0.20 µSv/h).

The "Background Compensation" menu item, on the other hand, serves to compensate for the counter tube's OWN zero effect. Larger counter tubes (e.g. from the VACUTEC company in Dresden, Germany) are made of glass, and glass contains potassium. So no matter how much you shield the counter tube with lead, it will still deliver pulses, albeit very few. To measure the dose rate accurately, you have to know this inherent effect of the counter tube and subtract it from the measured value. For a J614 counter tube, this value is irrelevant because it delivers far too few pulses for an accurate measurement in the nSv/h range.

**Reply by J0s3f** [reply]
Makes sense. 25 seemed quite high to me for background radiation. But apart from the FS-5000, I only have a fake GC-01 which shows much lower background radiation values, but also has a smaller tube and is not very trustworthy at all!
